resources/sass/global.scss

```diff
@@ -1257,7 +1257,7 @@ span.tooltip-info {
 }
 
 span.tooltip-info.left {
-  @extend span.tooltip-info;
+  @extend span, .tooltip-info;
```

Comment:
was this from error detection??

Reply:
Yep! There's a couple hundred deprecation warnings that still output when you do it, but I didn't address those.